includes docker-compose.yml for dev ease

kalmanchapman opened the pull request:
updates readme with docker-compose instructions

kalmanchapman committed:
includes docker-compose.yml for dev ease
updates readme with docker-compose instructions

gabe-ochoa commented on docker-compose.yml:
```diff
@@ -0,0 +1,19 @@
+  spark-worker:
+    build: .
+    command: /usr/local/bootstrap.sh worker
+    links:
```
@kalmanchapman let's open the worker webui port as well for introspection. I believe it's 4040 but check me on that.

kalmanchapman replied:
would be very handy - only issue could be when scaling locally with `docker-compose scale` the port may already attached to a previous container. is it possible to give a range?
or maybe make the workers available on nat?

kalmanchapman committed:
includes port exposure for worker node
updates readme to remove docker-compose scale suggestion (won't work
with bound port, and kinda overkill for dev) and info about where to
find worker web ui

kalmanchapman committed:
removes extra port I exposed

kalmanchapman committed:
just kidding, this should be exposed as well

kalmanchapman committed:
sorry to have gone back and forth on this twice - but I misread that
the job ui was on a worker ip. in fact, it is broadcast from the host
that submits the job - it isn't necessary, and can shadow the actual ui
if you're on local

kalmanchapman committed:
exposes worker ui ports without binding on host
updates README accordingly
